### PR TITLE
Hum 38 invalid status chip color on mobile

### PR DESCRIPTION
**Pull request by KacperKoza343**
task HUM-38

### CODE REVIEW

**Review by dnechay — COMMENTED** [review]
Good catch with "reject" button! I though it was intentionally disabled, but now it looks better

Please resolve merge conflict

One question: is there a plan to support dynamic colors per status (there are some in figma) or only main one?

**Comment by KacperKoza343** [comment]
> Good catch with "reject" button! I though it was intentionally disabled, but now it looks better
> 
> Please resolve merge conflict
> 
> One question: is there a plan to support dynamic colors per status (there are some in figma) or only main one?

I need to find a message where we discussed status of these colors, because I believe that one color for all states was a temporary solution that we choose. If we want to handle all this colors I need to have a union of available colors in response

File: packages/apps/human-app/frontend/src/pages/worker/jobs/components/my-jobs/mobile/my-jobs-table-mobile.tsx [hunk 1]
```diff
@@ -162,15 +172,27 @@ export function MyJobsTableMobile({
...
+                        <TableButton
+                          component={Link}
+                          disabled={buttonDisabled}
+                          fullWidth
+                          target="_blank"
+                          to={d.url}
+                        >
+                          {t('worker.jobs.solve')}
+                        </TableButton>
+                        <RejectButton
```
Comment (dnechay) [inline packages/apps/human-app/frontend/src/pages/worker/jobs/components/my-jobs/mobile/my-jobs-table-mobile.tsx:185]: @KacperKoza343 probably something went wrong while resolving conflicts and now we have 2 "reject" buttons

**Review by dnechay — CHANGES_REQUESTED** [review]

File: packages/apps/human-app/frontend/src/pages/worker/jobs/components/my-jobs/mobile/my-jobs-table-mobile.tsx (hunk 1 above)
Comment (KacperKoza343) [reply packages/apps/human-app/frontend/src/pages/worker/jobs/components/my-jobs/mobile/my-jobs-table-mobile.tsx:185]: done

**Review by dnechay — APPROVED** [review]

**Pull request merged**